addons/l10n_jo_edi/models/account_edi_xml_ubl_21_jo.py

```diff
@@ -103,8 +103,8 @@ def _get_country_vals(self, country):
 
     def _get_partner_party_identification_vals_list(self, partner):
         return [{
-            'id_attrs': {'schemeID': 'TN'},
-            'id': partner.vat,
+            'id_attrs': {'schemeID': 'TN' if partner.country_code == 'JO' else 'PN'},
+            'id': partner.vat or '',
```

Comment:
added that because otherwise the line in the xml is not present which is wrong, it should always be present even if empty

Reply:
And what are we supposed to put when the guy puts `/` as VAT number? It is the value we advise to put for people not subject to vat
(I think it's a generic issue for ubl btw, not specific to Jordania)

Reply:
i can ask yatin about that :smile: